src/modules/Bootloader.rb

```diff
@@ -204,7 +204,8 @@ def Summary(simple_mode: false)
       # kokso: additional warning that root partition is nfs type -> bootloader will not be installed
       if BootStorage.boot_filesystem.is?(:nfs)
         log.info "Bootloader::Summary() -> Boot partition is nfs type, bootloader will not be installed."
-        return _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        msg = _("The boot partition is of type NFS. Bootloader cannot be installed.")
```

Comment:
I would rephrase it like `"The boot partition is of type NFS. No bootloader is installed."` Or if you want to have it red and with warning, then I would write something like `"The boot partition is of type NFS. No bootloader is installed. Please ensure that your machine can properly boot from network."` So at least user have hint, but for me it is not needed, as if user want nfs root ( not trivial to setup ), then he knows what he is doing.

Reply:
Just used the same message, but also fine about rewording it.

Reply:
If the original message is already translated, then I wouldn't change it